src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/UpdateSentenceDateService.kt

```diff
@@ -170,6 +178,9 @@ class UpdateSentenceDateService(
         append("TUSSD ${sentenceDatesRequest.topupSupervisionStartDate} ")
         append("TUSED ${sentenceDatesRequest.topupSupervisionExpiryDate} ")
         append("PRRD ${sentenceDatesRequest.postRecallReleaseDate}")
+        if (licenceEntity is HdcLicence) {
+          append("HDCAD ${sentenceDatesRequest.homeDetentionCurfewActualDate} ")
+        }
```

Comment:
based on our discussion, this is all cool and good we're limiting the amount of type-checking to this specific case  👍